src/ert/ensemble_evaluator/_wait_for_evaluator.py

```diff
@@ -42,7 +42,7 @@ async def wait_for_evaluator(
     token: Optional[str] = None,
     cert: Optional[Union[str, bytes]] = None,
     healthcheck_endpoint: str = "/healthcheck",
-    timeout: Optional[float] = None,
+    timeout: Optional[float] = None,  # noqa: ASYNC109
```

Comment:
Can you add this exception in `pyproject.toml` instead, makes it easer to know that it is something to fix.

Reply:
No, just keep them as is, they all seems warranted enough.